deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvironBean.java

```diff
@@ -51,6 +51,7 @@
  * acc_type      VARCHAR(32)         NOT NULL,
  * email_recipients VARCHAR(1024),
  * watch_recipients VARCHAR(1024),
+ * group_mention_recipients VARCHAR(1024),
```

Comment:
why we need this?

Reply:
Sorry, it is because I missed this in one of my old PR. Since it is just a comment, I think we are OK. ^^

Reply:
it's better to have it in a separate PR to avoid different purpose PRs. It's also better for us if we want to deploy or revert the change.

Reply:
make sense. thanks